python/pyspark/sql/tests.py

```diff
@@ -3036,6 +3052,9 @@ def test_toPandas_arrow_toggle(self):
         pdf = df.toPandas()
         self.spark.conf.set("spark.sql.execution.arrow.enable", "true")
         pdf_arrow = df.toPandas()
+        # need to remove timezone for comparison
+        pdf_arrow["7_timestamp_t"] = \
+            pdf_arrow["7_timestamp_t"].apply(lambda ts: ts.tz_localize(None))
```

Comment:
@ueshin so you are saying that the timezone would only be added in Python when making the Pandas DataFrame?  Then Java would only use an Arrow timestamp without a timezone specified and treat it as timezone naive?